pkg/operator/ceph/file/filesystem_test.go

```diff
@@ -142,6 +142,7 @@ func fsExecutor(t *testing.T, fsName, configDir string, multiFS bool, createData
 	if multiFS {
 		return &exectest.MockExecutor{
 			MockExecuteCommandWithOutput: func(command string, args ...string) (string, error) {
+				fmt.Println(args[0:5])
```

Comment:
Was this just temporary? If keeping it, let's use the logger methods.
```suggestion
				logger.Info(args[0:5])
```

Reply:
it was for debugging, removed it.